indico/modules/events/reminders/templates/emails/_agenda.html

```diff
@@ -0,0 +1,33 @@
+{%- macro render_agenda(event, agenda) -%}
+<p><strong>{% trans title=event.title %}Agenda for {{ title }}{% endtrans %}</strong></p>
```

Comment:
Since this is an HTML email, you can indent things properly here w/o having to worry about plaintext indentations, blank lines, etc.